massa-execution-exports/src/controller_traits.rs

```diff
@@ -22,7 +22,11 @@ use std::collections::BTreeMap;
 use std::collections::HashMap;
 
 #[cfg(feature = "execution-trace")]
-use crate::types::{AbiTrace, SlotAbiCallStack, Transfer};
+use crate::types_trace_info::AbiTrace;
```

Comment:
Can we wrap them in one import like before ?

Reply:
Done